src/convertToObject.js

```diff
@@ -6,7 +6,30 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const result = {};
+
+  /* const newStr = sourceString.split(';');
+
+  for (const str of newStr) {
+    const [key, value] = str.split(':');
+
+    if (key && value) {
+      result[key.trim()] = value.trim();
+    }
+  } */
```

Comment:
The block of code from lines 15 to 23 is commented out and seems to be an alternative implementation of the logic. Since the same logic is implemented in lines 24 to 30, you can safely remove these commented-out lines to clean up the code.